bin/user/mqtt.py

```diff
@@ -193,7 +205,60 @@ def _obfuscate_password(url):
     'percent': None,
     'unix_epoch': None,
     }
-
+# The type of data delivered by a sensor, impacts how it is displayed in the frontend of HomeAssistant.
+# Each sensor can have defined device_class and/or unit_of_measurement passed in configuration
+# https://www.home-assistant.io/integrations/sensor/
+HA_SENSOR_TYPE = {
```

Comment:
HA_SENSOR_TYPE needs mm and mm_per_hour. I used a unit customisation to get rain in mm, but did not work until I added mm to thsi lookup. Same for HA_SENSOR_UNIT